### PR TITLE
Docs on becoming a listed user and carve out build to dedicated file

### LINKED ISSUE #551: We are looking for customers to be listed as a customer on keda.sh
We are looking for KEDA customers who are interested to be listed as a customer on keda.sh.

If you are, please comment here and provide a logo that we can use.

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Docs on becoming a listed user and carve out build to dedicated file.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Fixes #551

### CODE REVIEW

**Comment by zroubalik** [comment]
@tomkerkhove Static Check failed, you should call it a day and try once again tomorrow 😄

File: BUILD.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+Default value: `console`
+
+### Metrics Server logging
+
+Find `--v=0` argument in Operator Deployment section in `config/metrics-server/deployment.yaml` file, modify it's value and redeploy.
+
+Allowed values are `"0"` for info, `"4"` for debug, or an integer value greater than `0`, specified as string
+
+Default value: `"0"`
```
Comment (turbaszek) [inline BUILD.md:155]:
```suggestion
Default value: `"0"`

```

File: README.md [hunk 2]
```diff
@@ -92,128 +76,14 @@ If interested in contributing or participating in the direction of KEDA, you can
...
 
-### Metrics Server logging
-Find `--v=0` argument in Operator Deployment section in `config/metrics-server/deployment.yaml` file, modify it's value and redeploy.
+You can find the latest releases [here](https://github.com/kedacore/keda/releases).
 
-Allowed values are `"0"` for info, `"4"` for debug, or an integer value greater than `0`, specified as string
+## Contributing
 
-Default value: `"0"`
+You can find contributing guide [here](./CONTRIBUTING.md) or learn how to build & deploy KEDA locally [here](./BUILD.md).
```
Comment (turbaszek) [inline README.md:89]:
```suggestion
You can find contributing guide [here](./CONTRIBUTING.md) or learn how to build & deploy KEDA locally [here](./BUILD.md).

```

**Review by ahmelsayed — APPROVED** [review]

**Review by ahmelsayed — APPROVED** [review]
LGTM

**Pull request merged**